tensorrt_llm/evaluate/nemo_skills_eval.py

```
@@ -0,0 +1,1425 @@
+        with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+            s.settimeout(0.5)
+            return s.connect_ex((host, port)) == 0
+
+    if _up():
+        return None
+
+    import subprocess  # nosec B404
+    import sys
+    import time
+
+    proc = subprocess.Popen(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)